### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

File: src/app/components/RoundedBalance/index.tsx [hunk 1]
```diff
@@ -27,7 +26,7 @@ export const RoundedBalance: FC<RoundedBalanceProps> = ({
   compactLargeNumbers,
   scope,
   showSign,
-  ticker,
+  ticker = '',
```
Comment (lubej) [inline src/app/components/RoundedBalance/index.tsx:29]: Nit: I would prefer if it was `null`.

File: src/app/components/RoundedBalance/index.tsx [hunk 2]
```diff
@@ -94,7 +91,7 @@ export const RoundedBalance: FC<RoundedBalanceProps> = ({
       <Tooltip
         arrow
         placement="top"
-        title={t('common.valueInToken', { ...getPreciseNumberFormat(value), ticker: tickerName })}
+        title={t('common.valueInToken', { ...getPreciseNumberFormat(value), ticker: ticker })}
```
Comment (lubej) [inline src/app/components/RoundedBalance/index.tsx:94]: Shorthand `ticker: ticker` -> `ticker`? Or do you think it is less readable?

**Review by lubej — APPROVED** [review]

File: src/app/components/RoundedBalance/index.tsx (hunk 2 above)
Comment (lukaw3d) [reply src/app/components/RoundedBalance/index.tsx:94]: I'm not decided. In this case I think it is less readable, more confusable with a function parameter. Other cons:
- I get annoyed when I don't find field assignments when searching for ` <field>:`
  but ` <field>?:`, `['<field>']`, ` ["<field>"]` also break that
- Looks inconsistent if you later add any non-shorthand properties

But I love the shorthand in returns  https://github.com/oasisprotocol/wallet/blob/dc5b9373c3807fa78b563637003922ab718a5b35/src/vendors/oasisscan.ts#L137-L144 :shrug:

File: src/app/components/RoundedBalance/index.tsx (hunk 1 above)
Comment (lukaw3d) [reply src/app/components/RoundedBalance/index.tsx:29]: Not sure if that could break something. I'll keep this cleanup without logic changes

**Pull request merged**